src/Events/Integrations/Modules/Ticket_Emails/Email/RSVP.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * Class RSVP.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Integrations\Modules\Ticket_Emails
+ */
+
+namespace TEC\Events\Integrations\Modules\Ticket_Emails\Email;
+
+use TEC\Events\Integrations\Modules\Ticket_Emails\Emails as TEC_Email_Handler;
+use TEC\Events\Integrations\Modules\Ticket_Emails\Template as Template;
+use TEC\Tickets\Emails\Email\RSVP as RSVP_Email;
+use TEC\Tickets\Emails\Email_Abstract;
+
+/**
+ * Class RSVP.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Integrations\Modules\Ticket_Emails
+ */
+class RSVP {
+	/**
+	 * The option key for the Event calendar links.
+	 *
+	 * @see Email_Abstract::get_option_key() for option key format.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_add_event_links = 'tec-tickets-emails-rsvp-add-event-links';
+
+	/**
+	 * The option key for the Event calendar invite.
+	 *
+	 * @see Email_Abstract::get_option_key() for option key format.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_add_event_ics = 'tec-tickets-emails-rsvp-add-event-ics';
+
+	/**
+	 * Filter the email settings and add TEC specific settings.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<array<string,mixed>> $settings The email settings.
+	 *
+	 * @return array<array<string,mixed>> $settings The modified email settings.
+	 */
+	public function filter_tec_tickets_emails_rsvp_email_settings( $settings ) {
+
+		$settings[ self::$option_add_event_links ] = [
+			'type'            => 'toggle',
+			'label'           => esc_html__( 'Include "Add to calendar" links', 'the-events-calendar' ),
+			'tooltip'         => esc_html__( "Include links to add the event to the user's calendar.", 'the-events-calendar' ),
+			'default'         => true,
+			'validation_type' => 'boolean',
+		];
+
+		$settings[ self::$option_add_event_ics ] = [
+			'type'            => 'toggle',
+			'label'           => esc_html__( 'Attach Calendar Invites', 'the-events-calendar' ),
+			'tooltip'         => esc_html__( 'Attach calendar invites (.ics) to the RSVP email.', 'the-events-calendar' ),
+			'default'         => true,
+			'validation_type' => 'boolean',
+		];
+
+		return $settings;
+	}
+
+	/**
+	 * Filters the attachments for the RSVP Emails and maybe add the calendar ics file.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,string> $attachments  The attachments for the Tickets Emails.
+	 * @param string               $email_id     The email ID.
+	 * @param Email_Abstract       $email_class  The email class.
+	 *
+	 * @return array<string,string> The filtered attachments for the RSVP Emails.
+	 */
+	public function filter_tec_tickets_emails_rsvp_email_attachments( $attachments, $email_id, $email_class ) {
+		if ( ! $email_class->is_enabled() ) {
+			return $attachments;
+		}
+
+		$use_ticket_email = tribe_get_option( $email_class->get_option_key( 'use-ticket-email' ), false );
+
+		if ( ! empty( $use_ticket_email ) ) {
+			$email_class = tribe( TEC\Tickets\Emails\Email\Ticket::class );
```

Comment:
Can we use a `use` to simplify the class name.
`use TEC\Tickets\Emails\Email\Ticket;`